tests/ConnectionStringCreatingTest.java

```diff
@@ -0,0 +1,107 @@
+import connectionTool.connections.*;
```

Comment:
move test class into src/test and add package declaration at the beginning of the file - now it's not recognized as test by Intellij